### DISCUSSION TITLE
Intents on a discord selfbot

### DISCUSSION THREAD

**Discussion by Soliux** [Rapptz/discord.py#6435] [Q&A]
Basically I am wanting to do something on a discord selfbot but its requires the use of intents now I don't know if there is a way to even enable these on a normal discord user account, anyone know if this is possible?

**Comment by Nemika-Haj** [comment]
Self bots are against the discord ToS, so better not even make it at all.

**Reply by Soliux** [reply]
I completely understand that it is against ToS but the code I am wanting to grab all the users in a guild it is saying I need to use intents but when I then enabled them in my code and run the bot it asks me to verify my discord email address each time then it stops and says 403 forbidden access.

Is there no way around this as I kinda need to get this working.

**Comment by mesub7** [comment]
As a selfbot does not access the discord API as a normal bot should do, it does not require intents.

As Nemika said though (as well as the warning in the documentation), it is against the [Discord Terms of Service.](https://discord.com/terms)

**Reply by Soliux** [reply]
I completely understand that it is against ToS but the code I am wanting to grab all the users in a guild it is saying I need to use intents but when I then enabled them in my code and run the bot it asks me to verify my discord email address each time then it stops and says 403 forbidden access.

Is there no way around this as I kinda need to get this working.

**Reply by mesub7** [reply]
As it's against the ToS, we can't really help further. Sorry about that.

**Reply by Soliux** [reply]
Are you able to atleast tell me if its possible to enable intents on a user account? If not I dont want to waste my time looking

**Reply by mesub7** [reply]
As I said in the first post, user accounts do not need them. So no, it is not possible.

Why don't you use an actual bot instead?
